src/app/file-manager/reducers/downloadables.reducer.ts

```diff
@@ -11,6 +11,26 @@ export interface DownloadableState {
     items: Item[];
 }
 
+export class DownloadableStateImpl implements DownloadableState {
+    itms: Item[];
```

Comment:
It would be clearer to not have a getter for `items` and instead keep a field called `items`. In the case where we do have a name conflict, it looks like the convention is to name the field using an underscore, e.g. `_items`.